src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/pyproject.toml

```diff
@@ -53,7 +52,7 @@ dash-mantine-components = "^0.12.1"
 dash-uploader = "^0.6"
 {% elif cookiecutter.dash_ui == "awc" %}
 ansys-web-components-dash = {version = ">=0.0.dev4", source = "solutions-private-pypi"}
```

Comment:
should we remove this package?

Reply:
This package is needed for the AWC UI model